core-services/egov-user/src/main/java/org/egov/user/web/contract/UserRequest.java

```diff
@@ -44,10 +44,8 @@ public class UserRequest {
     @Size(max = 15)
     private String gender;
 
-    @Pattern(regexp = UserServiceConstants.PATTERN_MOBILE)
```

Comment:
just add the pattern variable here this will do auto-validation. add the same for an alternate mobile number and verify.

Reply:
We have moved the regex value to application properties and that can't be accessed here directly

Reply:
We have moved the regex value to application properties and that can't be accessed here directly